src/core/gui/PaperFormatUtils.cpp

```diff
@@ -0,0 +1,57 @@
+#include "PaperFormatUtils.h"
+
+#include "util/GListView.h"
+#include "util/StringUtils.h"
+#include "util/raii/GObjectSPtr.h"
+
+using xoj::util::GtkPaperSizeUPtr;
+
+void PaperFormatUtils::loadDefaultPaperSizes(PaperFormatMenuOptionVector& paperSizes) {
+    GList* default_sizes = gtk_paper_size_get_paper_sizes(false);
+    for (auto& s: GListView<GtkPaperSize>(default_sizes)) {
+        std::string name = gtk_paper_size_get_name(&s);
+        if (name == GTK_PAPER_NAME_A3 || name == GTK_PAPER_NAME_A4 || name == GTK_PAPER_NAME_A5 ||
+            name == GTK_PAPER_NAME_LETTER || name == GTK_PAPER_NAME_LEGAL) {
```

Comment:
To answer both questions: I adopted the selection of formats from the paper format dialog that is accessible through `Journal > Paper Format` and unified this loading for the dialog and the popover. I also think other users might want other formats, but we can't provide all formats out of the box:

## With if-statement:
<img src="https://github.com/user-attachments/assets/aaf1b4fa-98c2-4a8c-b299-a90812c7e536" width="200">

## Without if-statement:
<div>
<img src="https://github.com/user-attachments/assets/88da2bb3-01fe-466e-b7f2-967be9d7478e" width="200">
<img src="https://github.com/user-attachments/assets/97b4fc7a-3052-4802-84b9-2faeddd740e1" width="200">
</div>

The list contains some really exotic paper formats and seems almost endless. It would be best to create a setting for this, to let the user select the formats they need. But this would probably be something for another PR.